dropgz/pkg/embed/payload.go

```diff
@@ -86,6 +86,19 @@ func deploy(src, dest string) error {
 		return err
 	}
 	defer rc.Close()
+
+	if _, err := os.Stat(dest); err == nil || !os.IsNotExist(err) {
+		old_file_dest := dest + ".old"
+		os.RemoveAll(old_file_dest)
+		if err = os.Rename(dest, old_file_dest); err != nil {
+			return errors.Wrapf(err, "not able to rename the %s file to %s old", dest, old_file_dest)
+		}
+	}
+
```

Comment:
empty lines may be used to separate logical blocks, but only very sparingly. tighten this up, all you've done is make it so that i have to scroll my page sooner 🙁

Reply:
Sure thanks will add it to the next commit.